src/components/GetRooms.vue

```diff
@@ -36,30 +36,35 @@ const config = {
 const app = firebase.initializeApp(config);
 const auth = firebase.auth();
 const db = getFirestore(app);
+const roomsCol = collection(db, "rooms");
```

Comment:
I pulled `roomsCol` out so it's in scope for use by `getRooms` and `subscribeToRoomUpdates`

Reply:
Ok nice. I tried to do this before and got errors, but I'm sure I had other things going wrong